### PR TITLE
Add imageWidth and imageHeight parameters

### PR DESCRIPTION
**Pull request by gergo85**
#1233

### CODE REVIEW

File: modules/cms/formwidgets/mediafinder/assets/js/mediafinder.js [hunk 1]
```diff
@@ -43,9 +43,18 @@
 
         this.$el.one('dispose-control', this.proxy(this.dispose))
 
+        if (this.options.thumbnailWidth > 0) {
+            this.$el.css('maxWidth', this.options.thumbnailWidth + 'px')
+        }
+
+        else if (this.options.thumbnailHeight > 0) {
```
Comment (LukeTowers) [inline modules/cms/formwidgets/mediafinder/assets/js/mediafinder.js]: Why is this an elseif instead of a separate if statement?

**Comment by gergo85** [comment]
The imageWidth and imageHeight parameters of Media finder formwidget are not equivalent to Fileupload formwidget parameters.
In some cases the preview image is too big, therefore I want to set the maximum width or height of preview image.

**Comment by LukeTowers** [comment]
If they are not directly equivalent, then I think they should be named differently. Perhaps `previewImageWidth` and `previewImageHeight`?

**Comment by daftspunk** [comment]
LGTM

**Comment by daftspunk** [comment]
Please add documentation for this https://github.com/octobercms/docs

**Pull request merged**